.storybook/primitives-v7.css

```diff
@@ -1,2 +1 @@
-@import '@primer/css/dist/primitives.css';
-@import '@primer/css/dist/color-modes.css';
+@import 'https://unpkg.com/@primer/css@21.1.1/dist/color-modes.css';
```

Comment:
Do we need this import to unpkg because we're going to be using the newer version of primitives?

Reply:
I think its just easier to pull this from unpkg because its temporary and we rarely make releases in Primer CSS these days. I should probably make it `@latest` instead though..